core/scripts/cli/sponsor/transactionUtils.js

```diff
@@ -11,7 +11,7 @@ const submitTransaction = async (web3, submitFn, message) => {
   const { receipt } = await submitFn();
   style.spinnerWritingContracts.stop();
   const etherscanLink = etherscanBaseUrl + "/tx/" + receipt.transactionHash;
-  console.log("Done! Transaction link:", etherscanLink);
+  console.log("Transaction submitted! Transaction link:", etherscanLink);
```

Comment:
I think this language clarifies better that the transaction was sent, but not confirmed

Reply:
Yep, Allison pointed out that this was particularly confusing in the "request withdrawal" case, where the text made it sound like the withdrawal was done.